templates/backupninja.conf.erb

```diff
@@ -72,7 +72,13 @@ libdirectory = <%= scope.lookupvar('backupninja::params::libdirectory') %>
 usecolors = <% if scope.lookupvar('backupninja::params::usecolors') %>yes<% else %>no<% end %>
 
 # default value for 'when'
+<% if scope.lookupvar('backupninja::params::when').is_a?(Array) -%>
```

Comment:
I have tried every possible combination to do this by editing the module in the puppetmaster. I hace tried `.instance_of`, `respond_to('each')`, `unless it is a string`, but it never seems to enter inside this conditional.

Reply:
Solved by checking the variable type in the manifest instead of doing it in the template